### PR TITLE
Added logic for KtisisHyperboreia.cs dungeon level 87

### PR DESCRIPTION
**Pull request by KaiserBh**
Pain :(

### CODE REVIEW

**Review by TheManta — CHANGES_REQUESTED** [review]
Thanks for submitting both dungeons -- it looks good! I was able to run both as DPS, and I appreciate the sub-zone checks for boss mechanics + defining spell lists outside of `RunAsync()`.

Before I approve this, please fix the line-specific comments below and then resolve any StyleCop/.editorconfig stuff in Visual Studio's "Error List"/Rider's "Problems" tab. Fixing the specific items first will shrink the list to just a few style things, so definitely tackle those first.

File: Plugins/Trust/Dungeons/KtisisHyperboreia.cs [hunk 1]
```diff
@@ -13,12 +26,212 @@ public class KtisisHyperboreia : AbstractDungeon
...
+            if (stack.IsCasting())
+            {
+                sidestepPlugin.Enabled = false;
+                AvoidanceManager.RemoveAllAvoids(i => i.CanRun);
+                await MovementHelpers.GetClosestAlly.Follow();
+                sidestepPlugin.Enabled = true;
+            }
+
+            // Lyssa First Boss
+            if (WorldManager.SubZoneId == 3766)
```
[the remark this answers is not in the document]
Comment (KaiserBh) [reply Plugins/Trust/Dungeons/KtisisHyperboreia.cs:122]: image
seem to be an issue when checking these.

File: Plugins/Trust/Dungeons/TheAitiascope.cs [hunk 2]
```diff
@@ -16,9 +26,187 @@ public class TheAitiascope : AbstractDungeon
...
+            if (stack.IsCasting())
+            {
+                sidestepPlugin.Enabled = false;
+                AvoidanceManager.RemoveAllAvoids(i => i.CanRun);
+                await MovementHelpers.GetClosestAlly.Follow();
+                sidestepPlugin.Enabled = true;
+            }
+
+            // Livia The Undeterred
+            if (WorldManager.SubZoneId == 3992)
```
[the remark this answers is not in the document]
Comment (KaiserBh) [reply Plugins/Trust/Dungeons/TheAitiascope.cs:112]: Tried to change as many as I can but I am having issues regarding using the SubZoneId.EnumOftheZone.

**Comment by KaiserBh** [comment]
Tried to change as many as I can but I am having issues regarding using the SubZoneId.EnumOftheZone.

File: Plugins/Trust/Dungeons/KtisisHyperboreia.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheManta) [reply Plugins/Trust/Dungeons/KtisisHyperboreia.cs:119]: `WorldManager.SubZoneId` is `uint` and the enum `SubZoneId` is backed by `uint`, so it's 100% safe to do this so we can keep the nice sub-zone names:
```suggestion
            if (WorldManager.SubZoneId == 3766)
            if (WorldManager.SubZoneId == (unit)SubZoneId.FrozenSphere)
```

**Review by TheManta — APPROVED** [review]

**Comment by TheManta** [comment]
Awesome, thanks again for making changes and playing along with me figuring out GitHub code reviews <3

**Pull request merged**